src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/packaging-tools.props

```
@@ -1,5 +1,10 @@
 <Project>
 
+  <PropertyGroup>
+    <SigningToolsDirectory Condition="'$(SigningToolsDirectory)' != ''">$(RepoRoot)signing\</SigningToolsDirectory>
+    <LicenseFileLocation Condition="'$(LicenseFileLocation)' != ''">$(RepoRoot)LICENSE.TXT</LicenseFileLocation>
```

Maybe not exactly this name if it collides, but `File` being the suffix for a file path is the style here:

```suggestion
    <LicenseFile Condition="'$(LicenseFile)' != ''">$(RepoRoot)LICENSE.TXT</LicenseFile>
```